### PR TITLE
Fix some spelling mistakes

### PR DESCRIPTION
**Pull request by lucc**

### CODE REVIEW

File: alot/commands/thread.py [hunk 1]
```diff
@@ -731,7 +731,7 @@ def apply(self, ui):
         if self.shell:
             self.cmd = [' '.join(self.cmd)]
 
-        # do teh monkey
+        # do the monkey
```
Comment (pazz) [inline alot/commands/thread.py]: this one was actually on purpose:-)

Comment (lucc) [reply alot/commands/thread.py]: you approved, so should I change this back or just merge like it is now?

**Comment by pazz** [comment]
nonono, it's fine, go ahead. :)

**Comment by lucc** [comment]
Also I noticed that you merged master into fix/spelling on my fork (the base of this PR). Was that intentional?

**Comment by pazz** [comment]
no, i did that using the github interface by clicking on "resolve conflicts".. sorry about that. I will just go ahead and merge this then.

**Pull request merged**